setup.py

```diff
@@ -34,7 +34,7 @@
     platforms='any',
     install_requires=[
         'click>=6.6',
-        'pycrypto>=2.6.1',
+        'pycryptodome>=3.10.1',
         'six>=1.10.0'
```

Comment:
Use of `six` could be dropped (either here or in a follow up).

https://github.com/asottile/pyupgrade/ is useful for removing much of its use, plus modernising syntax for newer Python versions.

Reply:
Absolutely right @hugovk! I made that into it's own PR https://github.com/dispel/jak/pull/62/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7L38

Reply:
Also the pyupgrade tool looks interesting. If you want to run it on a branch and we can make sure it doesn't break anything then i'm all for sticking to python3 best practices for the code. I would recommend basing that branch on the `remove-compat` branch (unless that has been merged. I'm thinking I'll merge PRs 59-61 in the next few days.

Reply:
Sure! Please see PR https://github.com/dispel/jak/pull/63.